public/reorderSiteAwards.php

```diff
@@ -157,10 +157,14 @@ function handleSaveAllClick() {
 function postAllAwardsDisplayOrder(awards) {
     showStatusMessage('Updating...');
 
-    // We want each award to only take up a single variable in the form POST payload.
-    const compressedAwards = awards.map((award) => [award.type, award.data, award.extra, award.number].join(','));
+    // We want the awards to occupy limited space over the network.
+    const compressAwards = (award) => ([award.type, award.data, award.extra, award.number].join(','));
 
-    $.post('/request/user/update-site-awards.php', { awards: compressedAwards })
+    // Pass both sets of awards as CSVs rather than array elements.
+    const sortedAwards = awards.filter(award => award.number !== -1).map(compressAwards).join(',');
+    const hiddenAwards = awards.filter(award => award.number === -1).map(compressAwards).join(',');
```

Comment:
Recommend separating rows with something other than comma in case you need to eyeball the raw data. Maybe semicolon or pipe?

```
1,1542,1,0,1,1667,1,21,1,1471,1,23
```
vs
```
1,1542,1,0;1,1667,1,21;1,1471,1,23
```

Reply:
Good idea, I went with pipe (`|`) separation in https://github.com/RetroAchievements/RAWeb/pull/1483/commits/96f7845fd327f2b901f213d60c389397571407da.